src/azure-cli-core/azure/cli/core/extension/__init__.py

```diff
@@ -7,9 +7,11 @@
 import traceback
 import json
 import re
+import sys
+import pkginfo
 
 from azure.cli.core._config import GLOBAL_CONFIG_DIR, ENV_VAR_PREFIX
-
+from distutils.sysconfig import get_python_lib
```

Comment:
What if there is extension containing C code (non-pure Python), will the get_python_lib() work?

Reply:
On my Debian box cpython libraries are in dist-packages too, so I'm not expecting issues:

```
>>> from distutils.sysconfig import get_python_lib
>>> get_python_lib()
'/usr/lib/python3/dist-packages'
```
```
$ find /usr/lib/python3/dist-packages/ -name *.so -type f | wc -l
241
```

Also, what we are after is the azure-cli-extensions directory where we store the symlinks - the symlinks themselves can point to anywhere